lib/usd/ui/layerEditor/layerEditorWidget.cpp

```diff
@@ -112,14 +120,48 @@ void setupDefaultMenu(SessionState* in_sessionState, QMainWindow* in_parent)
         // the time to be created
         QObject::connect(createMenu, &QMenu::aboutToShow, in_parent, aboutToShowCallback);
 
-        auto optionMenu = menuBar->addMenu(StringResources::getAsQString(StringResources::kOption));
+        auto optionMenu
+            = menuBar->addMenu(StringResources::getAsQString(StringResources::kOptions));
         auto action = optionMenu->addAction(
             StringResources::getAsQString(StringResources::kAutoHideSessionLayer));
         QObject::connect(
             action, &QAction::toggled, in_sessionState, &SessionState::setAutoHideSessionLayer);
         action->setCheckable(true);
         action->setChecked(in_sessionState->autoHideSessionLayer());
 
+        auto usdSaveMenu = optionMenu->addMenu(
+            StringResources::getAsQString(StringResources::kUsdSaveFileFormat));
+        auto formatGroup = new QActionGroup(usdSaveMenu);
+        formatGroup->setExclusive(true);
+        auto formatBinary
+            = formatGroup->addAction(StringResources::getAsQString(StringResources::kBinary));
+        auto formatAscii
+            = formatGroup->addAction(StringResources::getAsQString(StringResources::kAscii));
+        auto grpAnn = StringResources::getAsQString(StringResources::kSaveLayerUsdFileFormatAnn);
+        auto grpSbm = StringResources::getAsQString(StringResources::kSaveLayerUsdFileFormatSbm);
+        formatBinary->setCheckable(true);
+        formatBinary->setData(1);
+        formatBinary->setToolTip(grpAnn);
+        formatBinary->setStatusTip(grpSbm);
+        formatAscii->setCheckable(true);
+        formatAscii->setData(0);
+        formatAscii->setToolTip(grpAnn);
+        formatAscii->setStatusTip(grpSbm);
+        usdSaveMenu->addAction(formatBinary);
+        usdSaveMenu->addAction(formatAscii);
+        bool isBinary = true;
+
+        auto formatArgsOption
+            = UsdMayaUtil::convert(MayaUsdOptionVars->mayaUsd_SaveLayerFormatArgBinaryOption);
+        if (MGlobal::optionVarExists(formatArgsOption)) {
+            isBinary = MGlobal::optionVarIntValue(formatArgsOption) != 0;
+        }
+        isBinary ? formatBinary->setChecked(true) : formatAscii->setChecked(true);
+        auto onFileFormatChanged = [=](QAction* action) {
+            MGlobal::setOptionVarValue(formatArgsOption, action->data().toInt());
+        };
+        QObject::connect(formatGroup, &QActionGroup::triggered, in_parent, onFileFormatChanged);
```

Comment:
This change is to add the binary or ascii option for .usd files to the Layer Editor menu.  It was previously in the file dialog itself.